admin/test/features/step_definitions/generate_pupil_pins_steps.rb

```diff
@@ -59,4 +59,28 @@
 And(/^I am on Generate pins Pupil List page$/) do
   step 'I navigate to generate pupil pins page'
   step 'I click Generate PINs button'
+  @page = generate_pupil_pins_page
```

Comment:
Why do we need an instance variable here ? @sinhaamit1980

Reply:
We need this for sticky banner steps which is common across Pupil check and Generate pin page